assessment.sql

```diff
@@ -5,24 +5,58 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
+CREATE TABLE licencees(
+  P_Id int NOT NULL,
+  surname CHARACTER VARYING(128),
+  given_name CHARACTER VARYING(128),
+  gender CHARACTER VARYING(1),
+  height INTEGER,
+  weight INTEGER,
+  dob DATE,
+  eye_color CHARACTER VARYING (10),
+  hair_color CHARACTER VARYING (10),
+  PRIMARY KEY (P_Id)
+);
```

Comment:
This answer suggests that you missed part of the lecture and compensated by finding answers online. Looking up information online is great and a valuable skill, but it doesn't substitute for understanding. In fact, understanding is required in order to know which answer is correct and how to alter it to suit your needs. 1/2